Screens/Walkthrough.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import { StyleSheet, Text, View, ScrollView, Image, StatusBar, FlatList, TouchableOpacity, TouchableNativeFeedback,
+  Button,
+  Platform } from 'react-native';
+import { DefaultTheme, Headline, Paragraph, TextInput,
+  Appbar, AppbarAction, AppbarBackAction, AppbarContent, AppbarHeader, 
+  Card, CardContent,
+  // Button,
+  ListAccordion, Divider,
+  ListSection, withTheme,
+  Toolbar, ToolbarBackAction, ToolbarContent, ToolbarAction, 
+  FAB, DrawerSection, Colors,
+  Provider as PaperProvider } from 'react-native-paper';
+import { createStackNavigator, DrawerActions } from 'react-navigation';
+import { Constants } from 'expo';
+import { RectangleButton, RoundButton } from '../Components';
+import Slideshow from 'react-native-slideshow';
+
+export default class Walkthrough extends React.Component {
+    static navigationOptions = ({ navigation }) => {
+      return {
+          header: null
+        };
+    };
+    constructor(props) {
+      super(props);
+      this.state = {
+        position: 1,
+        interval: null,
+        dataSource: [
```

Comment:
Rename to just say `data`. `dataSource={this.state.data}` is a very common thing.